src/server/lib/user-features.ts

```diff
@@ -42,3 +60,47 @@ export const setUserFeatureCookies = async ({
 		});
 	}
 };
+
+export const createCookie = ({
+	name,
+	res,
+	daysTillExpiry,
+}: {
+	name: string;
+	res: Response;
+	daysTillExpiry: number;
+}) => {
+	const tmpDate = new Date(Date.now() + daysTillExpiry * 24 * 60 * 60 * 1000);
+	res.cookie(name, tmpDate.getTime().toString(), {
+		domain,
+		expires: tmpDate,
+	});
+};
+
+export const persistUserBenefitsCookies = ({
```

Comment:
```suggestion
const persistUserBenefitsCookies = ({
```

Reply:
I think we should also add a comment to the corresponding secion in the DCR repo saying that if new cookies/benefits are added/removed there, that this section in gateway should also be updated:

https://github.com/guardian/dotcom-rendering/blame/a0eff55ee7fedf08b785557fceaa0f4f7265e0df/dotcom-rendering/src/client/userFeatures/user-features.ts#L38-L49